src/tensorlake/functions_sdk/functions.py

```diff
@@ -243,6 +244,14 @@ def construct(fn):
             "memory": memory,
             "ephemeral_disk": ephemeral_disk,
             "gpu": gpu,
+            "cache_key": (
```

Comment:
This approach slightly changes they way we were handling function attributes, the current approach for SDK function attributes is to either use basic data types like int, str, None or to define a struct like here https://github.com/tensorlakeai/tensorlake/blob/main/src/tensorlake/functions_sdk/functions.py#L84.
If we need to derive a value/struct from a function attribute then we do it at the place where this derived value is needed.

If you don't mind let's keep using it for consistency.

Reply:
Done -- I simplified it to a single string, since that's really all the server cares about.